jvm/ml4ir-inference/src/main/java/ml4ir/inference/tensorflow/SimpleExampleExecutor.java

```diff
@@ -0,0 +1,33 @@
+package ml4ir.inference.tensorflow;
+
+import com.google.common.collect.ImmutableMap;
+import ml4ir.inference.tensorflow.data.ModelFeaturesConfig;
+import ml4ir.inference.tensorflow.data.StringMapExampleBuilder;
+import org.tensorflow.example.Example;
+import java.util.Map;
+
+public class SimpleExampleExecutor {
+    private final ExampleExecutor executor;
+    private final StringMapExampleBuilder protoBuilder;
+    public SimpleExampleExecutor(String modelBundlePath,
+                                 String inputNodeName,
+                                 String outputNodeName,
+                                 String featureConfigPath) {
+        executor = new ExampleExecutor(
+                modelBundlePath,
+                ModelExecutorConfig.apply(inputNodeName, outputNodeName)
+        );
+        protoBuilder = StringMapExampleBuilder.withFeatureProcessors(
+                ModelFeaturesConfig.load(featureConfigPath),
+                ImmutableMap.of(),
+                ImmutableMap.of(),
+                ImmutableMap.of()
+        );
+    }
+
+    public float[] predict(Map<String, String> features) {
```

Comment:
Could we add a comment about what should be the keys/values of features? (with an example?)